### DISCUSSION TITLE
看看学信网，日本专区，只是不更新了。

### DISCUSSION THREAD

**Discussion by Sunovn** [The-Run-Philosophy-Organization/run#273] [Ideas]
有个日本留学专区，具体功能有没有用不清楚 反正都是十几年前的东西。我看了一些留学感想，原来当时是可以这样说的呀（最后一张图）。![Screenshot_20220426223812.jpg](https://user-images.githubusercontent.com/91693891/165331950-3fccfa4b-80dd-4fdd-a682-e818ad01a054.jpg)![Screenshot_2022-04-26-22-38-20-238_mark.via.jpg](https://user-images.githubusercontent.com/91693891/165332189-2467f574-c4fc-4019-a990-afd45283dd21.jpg)![Screenshot_2022-04-26-22-40-01-418_mark.via.jpg](https://user-images.githubusercontent.com/91693891/165332225-86c48d65-26bc-43eb-b9fb-3b501a0e68bd.jpg)![Screenshot_2022-04-26-22-40-12-981_mark.via.jpg](https://user-images.githubusercontent.com/91693891/165332245-ecd5bf0f-0887-44b0-aad3-55a40db5dc14.jpg)![Screenshot_2022-04-26-22-40-26-759_mark.via.jpg](https://user-images.githubusercontent.com/91693891/165332271-21cc7474-28cb-4125-b65e-823f9f7c8c63.jpg)

**Comment by Sunovn** [comment]
如果万玲玲的话放现在说，网络暴力就来了

**Reply by qibar** [reply]
起手式：得新冠死在外国吧
